components/script/dom/headers.rs

```diff
@@ -85,7 +85,7 @@ impl HeadersMethods for Headers {
         let mut combined_value: Vec<u8> = vec![];
         if let Some(v) = self.header_list.borrow().get_raw(&valid_name) {
```

Comment:
I highly suspect that we can do something fancy here, such as `filter_map` on the `header_list` iterator.